content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -18,7 +18,7 @@ A Kubernetes cluster (KubeVirt infrastructure cluster), which consists of nodes
 * 4GB of RAM
 * 30GB of storage.
 
-The cluster version must be in the scope of [supported KKP Kubernetes clusters](https://docs.kubermatic.com/kubermatic/v2.21/tutorials-howtos/operating-system-manager/compatibility/#kubernetes-versions)
+The cluster version must be in the scope of [supported KKP Kubernetes clusters](https://docs.kubermatic.com/kubermatic/main/tutorials-howtos/operating-system-manager/compatibility/#kubernetes-versions)
```

Comment:
Same here (I didn't notice this earlier). We don't want to link to a full URL when we can use references instead.

Reply:
fixed